Set/Game/AspectVGrid.swift

```diff
@@ -4,6 +4,7 @@ struct AspectVGrid<Item, ItemView>: View where ItemView: View, Item: Identifiabl
     var items: [Item]
     var aspectRatio: CGFloat
     var content: (Item) -> ItemView
+    var expectedCount = 12
```

Comment:
I'm quire sure you already have somewhere this number, where you decide how much card should be in the deck. At this moment it's hardcoded, and you will have to change it in two places when you wish to change initial number of card. I would recommend inject this value same place where you inject `items` itself.

Reply:
Please stop using `var`, do `let` by default 😐

Reply:
I don't have an idea how to fix it, so I created separate issue for this comment - https://github.com/mkotyak/stanford_cs193p_set_card_game/issues/14